ballerina/http_connection.bal

```diff
@@ -113,9 +113,11 @@ public client class Caller {
         return nativeGetRemoteHostName(self);
     }
 
-    private isolated function returnResponse(anydata|StatusCodeResponse|Response|error message, string? returnMediaType)
-            returns ListenerError? {
+    private isolated function returnResponse(anydata|StatusCodeResponse|Response|error message, string? returnMediaType,
+        HttpCacheConfig? cacheConfig) returns ListenerError? {
         Response response = new;
+        boolean setETag = cacheConfig is () ? false: cacheConfig.setETag;
+        boolean enableCache = false;
```

Comment:
```suggestion
        boolean cacheCompatibleType = false;
```
Is this more readable? So we know that, this flag is there to show the compatible types